src/components/SplitPane.tsx

```diff
@@ -133,6 +142,59 @@ export function SplitPane(props: SplitPaneProps) {
 
   const isFinalClosed = isParentClosed ? true : isSidePaneClosed;
 
+  const getSplitInnerPaneStyle = useCallback(
+    (side: SideSeparation) => {
+      return cssStyles.item(
+        isFinalClosed,
+        sideSeparation === side,
+        orientation,
+        size,
+        type,
+      );
+    },
+    [isFinalClosed, orientation, sideSeparation, size, type],
+  );
```

Comment:
There is no need to have a `useCallback` here. It created an additional layer of indirection which makes the code more difficult to read and understand

I think you can just call `cssStyles.item` directly and put the styles in a variable

Reply:
because it is called into two places based on a hardcoded sideSeparation value

Reply:
It's fine for me to keep it that way. But maybe we can remove the `useCallback` unless there is a performance issue linked to recreating this function on every render